hash_practice/exercises.py

```diff
@@ -5,16 +5,83 @@ def grouped_anagrams(strings):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+
+    def create_freq_table(string):
+        new_table = {}
+        new_table["word"] = string
+        for char in string:
+            if char in new_table:
+                new_table[char] += 1
+            else:
+                new_table[char] = 1
+        return new_table
+    def is_anagram(string, table):
+        table_copy = table.copy()
+        for char in string:
+            if char in table_copy:
+                if table_copy[char] > 0:
+                    table_copy[char] = table_copy[char] - 1
+                    continue
+                else:
+                    return False
+            else:
+                return False
+        return True

```

Comment:
I might add space here just for readability! Helps differentiate the inner function from the rest of the function body.

```suggestion
        return True
        
```